src/runner/provider/qti.js

```diff
@@ -65,6 +65,15 @@ var qtiItemRuntimeProvider = {
 
                 done();
             }, this.getLoadedClasses());
+
+            /**
+             * Get data related to QTI APIP accessibility
+             * @return {Object|null} accessibility data provided in test
+             */
+            self.getApipData = function getApipData() {
+                var apipData = self._item && _.isFunction(self._item.getApipAccessibility) && self._item.getApipAccessibility() || null;
```

Comment:
I don't really understand. You will also add a method `getApipAccessibility` on the loaded item. 
Can you show me the implementation ?
Because if the goal of this function is only to return the data from  `itemData.apipAccessibility` I'd simplify your approach : 
1. Expose a method `getData` on the item API 
2. This method returns the whole item data (is `itemData`)
3. In the implementation to access the `apip` data youll' have to do `itemRunner.getData().apipAccessibility` 
No ?

Reply:
"getApipAccessibility" was implemented by Tamas in QTI Item. So I add method "getApipData" to qti item runner (as reccomended by @jsconan https://github.com/oat-sa/tao-item-runner-qti-fe/pull/34#pullrequestreview-327538866) for  resolve task https://oat-sa.atlassian.net/browse/TAO-9637, aquiring of all data is excessively for it. Implementation of getApipAccesibility is here: https://github.com/oat-sa/tao-item-runner-qti-fe/blob/56e76ca96615167ab322e517a24f8c9f31df2d16/src/qtiItem/core/Item.js#L159